src/DevHome.csproj

```diff
@@ -45,7 +45,7 @@
         </PackageReference>
         <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
         <PackageReference Include="WinUIEx" Version="1.8.0" />
-        <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.2.8" GeneratePathProperty="true">
+        <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.4.0" GeneratePathProperty="true">

```

Comment:
IIRC we use powershell SDK for applying DSC configurations. Any idea if this is also needed elsewhere, or if the version bump is required by another package?